lib/prot/to2/msg61.c

```diff
@@ -189,31 +189,33 @@ int32_t msg61(fdo_prot_t *ps)
 	ret = -1; /* Reset to error */
 	LOG(LOG_DEBUG, "TO2.ProveOVHdr: Valid Ownership Header received\n");
 
-	ps->n5r = fdo_byte_array_alloc(FDO_NONCE_BYTES);
-	if (!ps->n5r) {
+	ps->nonce_to2proveov_rcv = fdo_byte_array_alloc(FDO_NONCE_BYTES);
+	if (!ps->nonce_to2proveov_rcv) {
 		goto err;
 	}
 	size_t nonce5_length = 0;
 	if (!fdor_string_length(&ps->fdor, &nonce5_length) || nonce5_length != FDO_NONCE_BYTES) {
-		LOG(LOG_ERROR, "TO2.ProveOVHdr: Invalid/Failed to read Nonce5 length\n");
+		LOG(LOG_ERROR, "TO2.ProveOVHdr: Invalid/Failed to read NonceTO2ProveOV length\n");
 		goto err;
 	}
-	if (!fdor_byte_string(&ps->fdor, ps->n5r->bytes, ps->n5r->byte_sz)) {
-		LOG(LOG_ERROR, "TO2.ProveOVHdr: Failed to read Nonce5\n");
+	if (!fdor_byte_string(&ps->fdor, ps->nonce_to2proveov_rcv->bytes,
+		ps->nonce_to2proveov_rcv->byte_sz)) {
+		LOG(LOG_ERROR, "TO2.ProveOVHdr: Failed to read NonceTO2ProveOV\n");
 		goto err;		
 	}
 
-	/* The nonces "n5" (msg40) and "n6" here must match */
-	if (!fdo_nonce_equal(ps->n5r, ps->n5)) {
-		LOG(LOG_ERROR, "TO2.ProveOVHdr: Received Nonce5 and Nonce5 do not match\n");
+	/* The nonces "NonceTO2ProveOV" from Type 60 and 61 must match */
+	if (!fdo_nonce_equal(ps->nonce_to2proveov_rcv, ps->nonce_to2proveov)) {
+		LOG(LOG_ERROR, "TO2.ProveOVHdr: Received NonceTO2ProveOV and NonceTO2ProveOV"
```

Comment:
Can we update this inline with msg/67 log?

`TO2.ProveOVHdr: Received NonceTO2ProveOV does not match with existing NonceTO2ProveOV`